packages/client/src/__tests__/types/namedConstraints/test.ts

```diff
@@ -23,6 +25,25 @@ async function main() {
         },
       },
     })
+
+  // @@unique
+  const result3 = await prisma.atAtUnique.findUnique({
```

Comment:
Should this be `AtAtUniqueNamed` maybe? THe `name` is on that model.

Reply:
Yes indeed

Reply:
I committed the changes to have the tests rerun